src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+function convertToNumber(currency) {
+  return Number(currency.split(',').join('.').slice(1));
+}
+
+thead.addEventListener('click', (e) => {
+  const title = e.target.textContent;
+  const index = e.target.cellIndex;
+
+  const rows = [...tbody.rows];
+  const enlargedRows = rows.map((row) => [row.cells[index].textContent, row]);
+
+  enlargedRows.sort((row1, row2) => {
```

Comment:
This is a code quality issue. The `sort` function modifies the original array. It's better to create a copy of the array before sorting, to keep the original data unchanged.